provider/cloudflare/cloudflare.go

```diff
@@ -43,6 +43,15 @@ const (
 	defaultCloudFlareRecordTTL = 1
 )
 
+// We have to use pointers to bools now, as the upstream cloudflare-go library requires them
+// see: https://github.com/cloudflare/cloudflare-go/pull/595
+
+// ProxyEnabled is a pointer to a bool true showing the record should be proxied through cloudflare
+var ProxyEnabled *bool = &[]bool{true}[0]
```

Comment:
tbh im not really happy about this message thing of creating a slice of bool, setting first element to true, and getting that element.

i wish there was a better way to do this, and am 500% all ears.